app/controllers/api/v1/monsters_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::MonstersController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
```

Comment:
Instead of needing to define this line for every API controller you might set up, you can create an `ApiController` that your API controllers can inherit from that includes this line. Your `ApiController` would instead inherit from the `ApplicationController` so you still gain access to the ActionController